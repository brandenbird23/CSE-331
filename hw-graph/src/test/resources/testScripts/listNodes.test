# listNodes test file
# tests if you can list nodes in multiple graphs

# Create Graphs
CreateGraph graph1
CreateGraph graph2

# Create Node
AddNode graph1 node1
AddNode graph2 node1

# Print the node in the graphs
ListNodes graph1
ListNodes graph2