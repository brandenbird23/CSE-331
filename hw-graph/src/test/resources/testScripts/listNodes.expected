# listNodes Test File
# tests if you can list nodes in multiple graphs

# Create graphs
created graph graph1
created graph graph2

# Create Node
added node node1 to graph1
added node node1 to graph2

# Print the node in the graphs
graph1 contains: node1
graph2 contains: node1